src/actors/common/augeas/magic.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python2
+
+import os
+import shlex
+import subprocess
+import uuid
+
+from pprint import pprint as pp
+
+import augeas
+
+## FIXME: this is tmp, make this as optional actor input
+INPUT = {"load": ["/opt/test_httpd.conf"],
```

Comment:
what is the purpose of this `INPUT` structure? IIUIC it specifies what additional files to `load`, which `directives` should be used for recursive scanning, and for which `lens` this recursive scanning should take place? How would I use this to add support for, e.g. postgresql `pg_hba.conf`?